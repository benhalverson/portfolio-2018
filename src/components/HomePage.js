import React from 'react';
const HomePage = () => (
<div className="content">
  <section className="hero">
    <img src="https://via.placeholder.com/200x200" alt="my pic" />
    <div className="hero-content">
      <h1>Title</h1>
      <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt
      magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue,
      in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate.
      Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum
      et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. 
      Curabitur ac quam sit amet velit cursus placerat.
       Phasellus ornare dui quis pharetra scelerisque.
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.</p>

<p>Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.</p>
      <a className="action-btn" href="/contact">Hire me</a>
    </div>
  </section>
</div>
)

export default HomePage;