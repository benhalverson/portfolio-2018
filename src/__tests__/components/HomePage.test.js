import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';
test('it should render the homepage', () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});
