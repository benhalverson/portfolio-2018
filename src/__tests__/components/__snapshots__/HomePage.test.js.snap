// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the homepage 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <section
        className="hero"
>
        <img
                alt="my pic"
                src="https://via.placeholder.com/200x200"
        />
        <div
                className="hero-content"
        >
                <h1>
                        Title
                </h1>
                <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
                </p>
                <p>
                        Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
                </p>
                <a
                        className="action-btn"
                        href="/contact"
                >
                        Hire me
                </a>
        </div>
</section>,
      "className": "content",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="my pic"
            src="https://via.placeholder.com/200x200"
/>,
          <div
            className="hero-content"
>
            <h1>
                        Title
            </h1>
            <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
            </p>
            <p>
                        Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
            </p>
            <a
                        className="action-btn"
                        href="/contact"
            >
                        Hire me
            </a>
</div>,
        ],
        "className": "hero",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "my pic",
            "src": "https://via.placeholder.com/200x200",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Title
</h1>,
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
</p>,
              <p>
                Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
</p>,
              <a
                className="action-btn"
                href="/contact"
>
                Hire me
</a>,
            ],
            "className": "hero-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Title",
              },
              "ref": null,
              "rendered": "Title",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.",
              },
              "ref": null,
              "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.",
              },
              "ref": null,
              "rendered": "Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Hire me",
                "className": "action-btn",
                "href": "/contact",
              },
              "ref": null,
              "rendered": "Hire me",
              "type": "a",
            },
          ],
          "type": "div",
        },
      ],
      "type": "section",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <section
          className="hero"
>
          <img
                    alt="my pic"
                    src="https://via.placeholder.com/200x200"
          />
          <div
                    className="hero-content"
          >
                    <h1>
                              Title
                    </h1>
                    <p>
                              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
                    </p>
                    <p>
                              Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
                    </p>
                    <a
                              className="action-btn"
                              href="/contact"
                    >
                              Hire me
                    </a>
          </div>
</section>,
        "className": "content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="my pic"
              src="https://via.placeholder.com/200x200"
/>,
            <div
              className="hero-content"
>
              <h1>
                            Title
              </h1>
              <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
              </p>
              <p>
                            Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
              </p>
              <a
                            className="action-btn"
                            href="/contact"
              >
                            Hire me
              </a>
</div>,
          ],
          "className": "hero",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "my pic",
              "src": "https://via.placeholder.com/200x200",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1>
                  Title
</h1>,
                <p>
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.
</p>,
                <p>
                  Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.
</p>,
                <a
                  className="action-btn"
                  href="/contact"
>
                  Hire me
</a>,
              ],
              "className": "hero-content",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Title",
                },
                "ref": null,
                "rendered": "Title",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.",
                },
                "ref": null,
                "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt magna turpis, rutrum sagittis nibh tincidunt non. Aenean egestas odio augue, in rutrum elit dictum sed. Curabitur bibendum ex ac ligula hendrerit vulputate. Suspendisse potenti. Donec tortor diam, rutrum sed suscipit eget, condimentum et est. Suspendisse potenti. Fusce ac nibh malesuada nulla efficitur ullamcorper. Curabitur ac quam sit amet velit cursus placerat. Phasellus ornare dui quis pharetra scelerisque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel posuere lectus. Mauris gravida, mauris ut scelerisque imperdiet, libero tellus mollis ipsum, quis interdum ipsum turpis in arcu.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.",
                },
                "ref": null,
                "rendered": "Maecenas id urna mattis nisi volutpat semper vitae sit amet justo. Phasellus pharetra ipsum sit amet neque imperdiet, non dictum nibh porta. Integer placerat nec nisl vel ultrices. Vivamus faucibus consequat velit. Curabitur eleifend justo ut lobortis bibendum. Mauris maximus nisi ultrices rhoncus pellentesque. Donec non interdum nisl. Nam facilisis felis non nibh vulputate, commodo consectetur tortor rutrum.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Hire me",
                  "className": "action-btn",
                  "href": "/contact",
                },
                "ref": null,
                "rendered": "Hire me",
                "type": "a",
              },
            ],
            "type": "div",
          },
        ],
        "type": "section",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
